docs/apm/traces/services-list-map.md

```diff
@@ -95,6 +116,9 @@ Manual anomaly detection allows you to specify different detection settings for
 
 <img src={useBaseUrl('img/traces/services-list-map-manual-detection-settings.png')} alt="services manual anomaly detection settings" width="400"/>
 
+### Alerts
+
+**Alerts** allows you to consider all associated alerts, which are currently active or resolved within the selected time.
```

Comment:
```
If this option is enabled, for a given service its status will be influenced by the presence of active alerts (or those, which have been resolved within the selected time range) associated with the service.
```
WDYT?

Reply:
Even this looks good to me. I will add this.